content/en/docs/reference/glossary/dataplane-mode.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Data Plane Mode
+test: n/a
+---
+
+The data plane mode refers to which mode the [data plane](/docs/reference/glossary/#data-plane) is running in.
```

Comment:
Istio currently supports three modes: [sidecar](/docs/reference/glossary/#sidecar), [ambient](/docs/reference/glossary/#ambient), and [proxyless](/docs/reference/glossary/#proxyless).